aten/src/ATen/native/mkldnn/Linear.cpp

```diff
@@ -260,6 +270,14 @@ Tensor mkldnn_linear_pointwise_binary(
     const Tensor& weight_t,
     const std::optional<Tensor>& bias_opt,
     std::string_view attr) {
+  auto aprop_kind = ideep::prop_kind::forward;
+  bool maybe_backward = GradMode::is_enabled() &&
+      (input_t.requires_grad() || weight_t.requires_grad() ||
+       (bias_opt.has_value() && bias_opt->defined() &&
+        bias_opt->requires_grad()));
+  if (!maybe_backward) {
+    aprop_kind = ideep::prop_kind::forward_inference;
+  }
```

Comment:
define a helper method for this repeated code block

Reply:
I modified the code, I directly set `aprop_kind ` in `mkldnn_linear_pointwise_binary ` function to `forward_inference ` since binary fusion is only for inference. Could you please review it again? Thanks!